om-2.0.rdf

```diff
@@ -38747,42 +38749,109 @@
+    <om:alternativeLabel xml:lang="en">CO2 equivalent emissions</om:alternativeLabel>
+    <rdfs:comment xml:lang="en">Global warming potential is an index of the total energy added to the climate system by the phenomenon relative to the total energy added by CO2 for a time horizon of 20 years.</rdfs:comment>
+    <rdfs:subClassOf rdf:resource="&om;GlobalWarmingPotential"/>
+    <om:symbol>GWP20</om:symbol>
+    <ombibo:reference rdf:resource="http://www.wikidata.org/entity/Q28324305"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="&om;GlobalWarmingPotential100YearHorizon">
+    <rdfs:label xml:lang="en">global warming potential for a time horizon of 100</rdfs:label>
+    <om:alternativeLabel xml:lang="en">CO2 equivalent emissions</om:alternativeLabel>
+    <rdfs:comment xml:lang="en">Global warming potential is an index of the total energy added to the climate system by the phenomenon relative to the total energy added by CO2 for a time horizon of 100 years. It has been determined as the standard metric for reporting emissions of various greenhouse gases relative to CO2 emissions by a decision in the Kyoto Protocol.</rdfs:comment>
+    <rdfs:subClassOf rdf:resource="&om;GlobalWarmingPotential"/>
+    <om:symbol>GWP100</om:symbol>
+    <ombibo:reference rdf:resource="http://www.wikidata.org/entity/Q28324305"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="&om;GlobalWarmingPotential500YearHorizon">
+    <rdfs:label xml:lang="en">global warming potential for a time horizon of 500</rdfs:label>
+    <om:alternativeLabel xml:lang="en">CO2 equivalent emissions</om:alternativeLabel>
+    <rdfs:comment xml:lang="en">Global warming potential is an index of the total energy added to the climate system by the phenomenon relative to the total energy added by CO2 for a time horizon of 500 years.</rdfs:comment>
+    <rdfs:subClassOf rdf:resource="&om;GlobalWarmingPotential"/>
+    <om:symbol>GWP500</om:symbol>
+    <ombibo:reference rdf:resource="http://www.wikidata.org/entity/Q28324305"/>
+  </owl:Class>
+
   <!-- Unit of Measure Ontologies -->
 
   <!-- Contents -->
 
-  <!-- Gram of Carbon Dioxide Equivalent Multiples and Submultiples Ontology -->
-  <!-- Gram of Carbon Dioxide Equivalent Per Watt Hour Multiples and Submultiples Ontology -->
+  <!-- Carbon Dioxide Equivalent Ontology -->
+  <!-- Gram Carbon Dioxide Equivalent Multiples and Submultiples Ontology -->
+  <!-- Gram Carbon Dioxide Equivalent Per Watt Hour Multiples and Submultiples Ontology -->
 
-  <!-- Gram of Carbon Dioxide Equivalent Multiples and Submultiples Ontology -->
+  <!-- Carbon Dioxide Equivalent Ontology -->
 
-  <om:Unit rdf:about="&om;gramOfCarbonDioxideEquivalent">
-    <rdfs:label xml:lang="en">gram of carbon dioxide equivalent</rdfs:label>
-    <om:symbol>gCO2eq</om:symbol>
+  <om:Unit rdf:about="&om;carbonDioxideEquivalent">
+    <rdfs:label xml:lang="en">carbon dioxide equivalent</rdfs:label>
+    <om:symbol>CO2eq</om:symbol>
     <rdf:type rdf:resource="&om;SingularUnit"/>

```

Comment:
`om:carbonDioxideEquivalent` (I think) needs to be defined based as `(W m⁻² yr kg⁻¹) / (W m⁻² yr kg⁻¹)`.

Reply:
`CO2-eq` is ambiguous, as it could be used for GWP (`CO2-eq = (W m⁻² yr kg⁻¹) / (W m⁻² yr kg⁻¹)`), GTP (`CO2-eq = (K kg⁻¹) / (K kg⁻¹)`), and probably more. I am not sure how to handle this. Is see two options:
1. Make multiple `CO2-eq` unit, or
2. create one ambiguous `CO2-eq`, as done e.g. for `ppm` too.

I now gave option 1 (multiple units) a try in commit 27d07e9. (But without replicating all units derived from `CO2-eq`)

@HajoRijgersberg What do you think about it?